jetty-websocket/websocket-javax-common/src/main/java/org/eclipse/jetty/websocket/javax/common/JavaxWebSocketFrameHandler.java

```diff
@@ -374,7 +374,7 @@ private void assertBasicTypeNotRegistered(byte basicWebSocketType, Object messag
         try
         {
             // TODO: move methodhandle lookup to container?
-            MethodHandles.Lookup lookup = MethodHandles.publicLookup();
+            MethodHandles.Lookup lookup = JavaxWebSocketFrameHandlerFactory.getServerMethodHandleLookup();
```

Comment:
Do you still need the TODO above?

Reply:
I think we can remove it, I don't know why we would want to do this in the container, it seems better to contain the MethodHandle stuff in the FrameHandler.